### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by simonhull** [DeviantJS/angular2-seed-ng2-bootstrap#3]
So this is solved using ng2-bootstrap.  But I'm wondering if there are any examples of Bootstrap being included as a standard CSS Javascript asset.  In my experience, I haven't been able to utilize any external javascript inside components.  And I know on the previous issue another guy was having trouble getting the Foundation library working as well.  Just wondering if there's a more inclusive fix that doesn't rely on specially made Angular 2 plugins.

Appreciate your time building this though!

**Comment by d3viant0ne** [comment]
If you are talking about using straight bootstrap 4, it's next on the list. I'm going to try and tackle that example repo this evening.

https://github.com/DeviantJS/angular2-seed-bootstrap4

I've done some reading on the issues people have been having, it shouldn't be a problem.

**Comment by simonhull** [comment]
Yeah I'd just like to see an example of a vanilla bootstrap/foundation build that doesn't rely on custom Angular components to work.  Looking forward to seeing your example, and thanks again for all the time you're putting into this!

**Comment by d3viant0ne** [comment]
No problem
